src/renderer/domains/collectives/model/voting/model.ts

```diff
@@ -47,19 +47,19 @@ const {
 
       if (nullable(accountId) || nullable(referendumId)) continue;
 
-      switch (vote.type) {
+      switch (vote.vote?.type) {
         case 'Aye':
           result.push({
             accountId,
```

Comment:
I think now you can map `accountId` and `referendumId` from vote itself, not by index